tests/__sf_section_docker.test.fish

```diff
@@ -0,0 +1,219 @@
+		set_color --bold fff
+		echo -n " "
+		set_color normal
+	) = (__sf_section_docker)
+end
+
+test "Prints section when only Dockerfile is present with DOCKER_MACHINE_NAME set"
+	(
+		rm /tmp/some-compose-file.yml
+		touch Dockerfile
+		set -g DOCKER_MACHINE_NAME some-machine-name
+
+		set_color --bold fff
+		echo -n "is "
+		set_color normal
+		set_color --bold cyan
+		echo -n "🐳 v$LOCAL_DOCKER_VERSION via $DOCKER_MACHINE_NAME"
+		set_color normal
+		set_color --bold fff
+		echo -n " "
+		set_color normal
+	) = (__sf_section_docker)
+end
+
+test "Prints section when only docker-compose.yml is present with DOCKER_MACHINE_NAME set"
+	(
+		touch docker-compose.yml
+		set -g DOCKER_MACHINE_NAME some-machine-name
+
+		set_color --bold fff
+		echo -n "is "
+		set_color normal
+		set_color --bold cyan
+		echo -n "🐳 v$LOCAL_DOCKER_VERSION via $DOCKER_MACHINE_NAME"
+		set_color normal
+		set_color --bold fff
+		echo -n " "
+		set_color normal
+	) = (__sf_section_docker)
+end
+
+test "Prints section when both Dockerfile and docker-compose.yml are present with DOCKER_MACHINE_NAME set"
+	(
+		touch Dockerfile
+		touch docker-compose.yml
+		set -g DOCKER_MACHINE_NAME some-machine-name
+
+		set_color --bold fff
+		echo -n "is "
+		set_color normal
+		set_color --bold cyan
+		echo -n "🐳 v$LOCAL_DOCKER_VERSION via $DOCKER_MACHINE_NAME"
+		set_color normal
+		set_color --bold fff
+		echo -n " "
+		set_color normal
+	) = (__sf_section_docker)
+end
+
+test "Prints Docker section when COMPOSE_FILE is set with DOCKER_MACHINE_NAME set"
+    (
+		set -g COMPOSE_FILE /tmp/some-compose-file.yml
+		touch /tmp/some-compose-file.yml
+		set -g DOCKER_MACHINE_NAME some-machine-name
+
+		set_color --bold fff
+		echo -n "is "
+		set_color normal
+		set_color --bold cyan
+		echo -n "🐳 v$LOCAL_DOCKER_VERSION via $DOCKER_MACHINE_NAME"
+		set_color normal
+		set_color --bold fff
+		echo -n " "
+		set_color normal
+	) = (__sf_section_docker)
+end
+
+test "Changing SPACEFISH_DOCKER_SYMBOL changes the displayed character"
+	(
+		rm /tmp/some-compose-file.yml
+		set SPACEFISH_DOCKER_SYMBOL "· "
+		touch Dockerfile
+
+		set_color --bold fff
+		echo -n "is "
+		set_color normal
+		set_color --bold cyan
+		echo -n "· v$LOCAL_DOCKER_VERSION"
+		set_color normal
+		set_color --bold fff
+		echo -n " "
+		set_color normal
+	) = (__sf_section_docker)
+end
+
+test "Changing SPACEFISH_DOCKER_PREFIX changes the character prefix"
+	(
+		set sf_exit_code 0
+		set SPACEFISH_DOCKER_PREFIX ·
+		touch Dockerfile
+
+		set_color --bold fff
+		echo -n "·"
+		set_color normal
+		set_color --bold cyan
+		echo -n "🐳 v$LOCAL_DOCKER_VERSION"
+		set_color normal
+		set_color --bold fff
+		echo -n " "
+		set_color normal
+	) = (__sf_section_docker)
+end
+
+
+# Negative
+test "Doesn't display section when SPACEFISH_DOCKER_SHOW is set to 'false'"
+	(
+		set SPACEFISH_DOCKER_SHOW false
+		touch Dockerfile
+
+	) = (__sf_section_docker)
+end
+
+test "Doesn't print section if docker is not installed"
+    (
```

Comment:
Looks like there is some inconsistent whitespace. Please double-check that your editor is using tabs here 👍 

```suggestion
	(
```

Reply:
I can't figure out how is this happening. 🤔